website/content/es/events/2024-cloud-native-sustainability-week.md

```diff
@@ -0,0 +1,50 @@
+---
+title: CNCF Cloud Native Sustainability Week 2024
+linkTitle: Cloud Native Sustainability Week 2024
+exclude_search: true
+description: Cloud Native Sustainability Week is a global event where the CNCF community organizes local meetings around the theme of Cloud Native Sustainability. The Cloud Native Sustainability Week will take place at the beginning week of October 2024.
+weight: 94
+slug: cloud-native-sustainability-week
+aliases:
+  - /cloud-native-sustainability-week
+---
+
+<p class="mt-5 mb-5"><img src="/images/cloud-native-sustainability-week-banner.webp" alt="CNCF Cloud Native Sustainability Week Banner"></p>
+
+The CNCF Sustainability Week is an annual event that is organized by the [CNCF Technical Advisory Group for Environmental Sustainability (TAG ENV)](http://github.com/cncf/tag-env-sustainability) in collaboration with the [Cloud Native Computing Foundation (CNCF)](http://cncf.io). The main goal of the event is to promote environmental sustainability in the cloud native industry and open source space.
+
+This global event, which includes both in-person meet-ups and virtual sessions, is organized by local tech communities around the world and coordinated by the TAG ENV. We believe that by encouraging such global intercontinental collaboration, we can enable people from all over the globe to understand how sustainability is an integral factor in the technology sector and how, sometimes with very little, technological and innovative initiatives can be carried out while respecting the environment.
+
+After the successful [Sustainability Week 2023](https://tag-env-sustainability.cncf.io/events/2023-cloud-native-sustainability-week/), planning for this year's edition is already underway, with a renewed commitment to increasingly green and engaging initiatives.
+
+**At the beginning week of October 2024**, the [CNCF community groups around the globe](https://community.cncf.io/chapters/) will organize meetups in their cities with a focus on cloud native sustainability. These meetups can be organized by anyone interested in supporting the effort. Additionally, the [CNCF Environmental Sustainability Technical Advisory Group (TAG ENV)](http://github.com/cncf/tag-env-sustainability), will host a virtual meetup. The recordings of the virtual and recorded in person event will be uploaded to the playlist on YouTube.
+
+<center><iframe width="600" height="400" src="https://www.youtube.com/embed/ezGSYtvQH2c?si=Qz5inM2pI4rHuerm" title="Sustainability Week 2024" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></center>
```

Comment:
can we(@donaldliu) have source of this video to make it on China social media?

Reply:
Hey @SamYuan1990 Sure, I will share the source video with you on slack. Thank you!